version: '3.8'

services:
 

  jupyter:
    image: jupyter/datascience-notebook:latest
    container_name: jupyter_python
    user: root
    environment:
      GRANT_SUDO: "yes"
      NB_GID: "100"
      JUPYTER_ENABLE_LAB: "yes"
      TZ: America/Sao_Paulo
    ports:
      - "8888:8888"
    volumes:
#      - $PWD/jupyter_data:/home/jovyan/work
      - $PWD:/home/jovyan/work
      - /etc/localtime:/etc/localtime:ro
      - /etc/timezone:/etc/timezone:ro
    command: start-notebook.sh --NotebookApp.token='python@2024'
    networks:
      - iot_network

networks:
  iot_network:
    driver: bridge
